tyr/policies/stackdriver_conditions.json

```diff
@@ -0,0 +1,198 @@
+{ 
+    "_disk_gt_70pct_10m": {
```

Comment:
Sure, if we want it to work that way.  I intended to make it easy to see which servers already have a condition and which don't (so we could then create the alert for that condition automatically where required).  

In reality it's been a bit trickier because we are using several different metric types (like cloudwatch, winagent, linux agent) and the JSON objects don't match easily.  But we should be able to make it more consistent, and then add new conditions to the config file and apply them as we like.